deepchem/models/tensorgraph/layers.py

```diff
@@ -424,18 +469,21 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
       parent = tf.expand_dims(parent, 2)
     elif len(parent.get_shape()) != 3:
       raise ValueError("Parent tensor must be (batch, width, channel)")
-    parent_shape = parent.get_shape()
-    parent_channel_size = parent_shape[2].value
-    f = tf.Variable(self.weights_initializer()(
-        [self.width, parent_channel_size, self.out_channels]))
-    t = tf.nn.conv1d(parent, f, stride=self.stride, padding=self.padding)
-    if self.biases_initializer is not None:
-      b = tf.Variable(self.biases_initializer()([self.out_channels]))
-      t = tf.nn.bias_add(t, b)
-    if self.activation_fn is None:
-      out_tensor = t
-    else:
-      out_tensor = self.activation_fn(t)
+    out_tensor = tf.keras.layers.Conv1D(
```

Comment:
Nice swap for keras here! We should make these swaps whenever we get the chance.